calico/maintenance/monitor/monitor-component-metrics.md

```diff
@@ -41,24 +60,40 @@ This tutorial will go through the necessary steps to implement basic monitoring
 3. Deploy and configure Prometheus.
 4. View the metrics in the Prometheus dashboard and create a simple graph.
 
-> **Note**: This guide assumes {{site.prodname}} was installed in the kube-system namespace. If using the Operator, you should replace instances of kube-system with calico-system instead.
-   {: .alert .alert-info}
-
-#### 1. Configure Calico to enable metrics reporting
+#### 1. Configure {{site.prodname}} to enable metrics reporting
 
 ##### **Felix configuration**
-Felix prometheus metrics are **disabled** by default. You have to manually change your Felix configuration (**prometheusMetricsEnabled**) via calicoctl in order to use this feature.
-
+Felix prometheus metrics are **disabled** by default.
 > **Note**: A comprehensive list of configuration values can be [found at this link]({{ site.baseurl }}/reference/felix/configuration).
    {: .alert .alert-info}
 
+Use the following command to enable Felix metrics.
+{% tabs %}
+<label:kubectl,active:true>
+<% 
+```bash
+kubectl patch felixconfiguration default --type merge --patch '{"spec":{"prometheusMetricsEnabled": true}}'
+```
+
+You should see an output like below:
+```
+felixconfiguration.projectcalico.org/default patched (no change)
```

Comment:
"no change" is wrong, since we're assuming the user hasn't done this yet. Can just drop the no change part.